### ISSUE TITLE
WEB: Follow us buttons in the pandas home page not displaying

### ISSUE THREAD

**Issue by datapythonista** [pandas-dev/pandas#57165]
In the section `Follow us` in https://pandas.pydata.org/ we should have links our social media. But seems like the JavaScript we link from Font Awesome is returning a 403 Forbidden, and the links are broken. Not sure why the link that used to work now it's not, we should do some research, and make the JavaScript load correctly so the icons are properly displayed.

The header of the website with the link to the JavaScript is in `web/_templates/layout.html`.

**Comment by koushik-rout-samsung** [comment]
take

**Comment by remiBoudreau** [comment]
@koushik-rout-samsung let me know if you need help pointing you in the right direction.

**Comment by koushik-rout-samsung** [comment]
@datapythonista @remiBoudreau I was able to solve it by updating the kit from fontawesome. I have used my own kit, is it advisable or there should be some common kit for using ? Also i have tested it in live browser, do i need the development setup for web if yes share the steps.

**Comment by datapythonista** [comment]
Not sure what do you mean by your own kit. Ideally we'd want to continue a CDN for the javacsript, so we don't need to add the file to the pandas repo. But if there are no reliable CDNs for this (I'm pretty sure there are), we can also add the file.

Feel free to open a pull request and tag me, so I can have a look at what you did. Thanks!

**Comment by koushik-rout-samsung** [comment]
@datapythonista actually the code is referring to some script which have changed the policies for usage. I will open a PR for this. 
https://github.com/FortAwesome/Font-Awesome/issues/17645

**Comment by datapythonista** [comment]
Ah, thank you, I see what you mean now, thanks for sharing that link.

I think this is more complex than initially expected. I created a kit using pandas credentials, but I see that the free plan comes with 10,000 pageviews per month, which is far from enough for pandas. I can't easily check how many pageviews we've got in the website, but only the home page has 350k/month, so not clear if the $99/year plan is enough. And not sure if worth paying just for couple of social media icons in the website. I guess we may want to use another alternative.
